fe/fe-core/src/main/java/com/starrocks/common/proc/LoadProcDir.java

```diff
@@ -52,7 +52,7 @@ public class LoadProcDir implements ProcDirInterface {
             .add("Type").add("Priority").add("ScanRows").add("FilteredRows").add("UnselectedRows")
             .add("SinkRows").add("EtlInfo").add("TaskInfo").add("ErrorMsg").add("CreateTime")
             .add("EtlStartTime").add("EtlFinishTime").add("LoadStartTime").add("LoadFinishTime")
-            .add("TrackingSQL").add("JobDetails")
+            .add("TrackingSQL").add("JobDetails").add("Warehouse")
```

Comment:
‌‌I suggest that we should maintain consistency with the previous logic, and add the display of this column when the cluster is running in shared-data mode.